src/NGen.cpp

```diff
@@ -1,5 +1,6 @@
 #include <iostream>
 #include <fstream>
+#include <pybind11/embed.h>
```

Comment:
Not supposed to be here, accident 🤦🏾

Reply:
Resolved [cd2fc7f](https://github.com/NOAA-OWP/ngen/pull/679/commits/cd2fc7fece0ccfdf5884a5cb7897af7e4172704c).